fullstack-mpp/build.gradle.kts

```diff
@@ -0,0 +1,94 @@
+import org.jetbrains.kotlin.gradle.targets.js.webpack.*
+
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:2.1.0")
+    }
+}
```

Comment:
This block can be removed from here as you've already specified repositories in the `settings.gradle.kts`